src/templates/blog-tag-list.js

```diff
@@ -7,7 +7,7 @@ import { graphql } from "gatsby";
 
 export const query = graphql`query BlogsByTags($tag: String!) {
   allMdx(
-    sort: {fields: [frontmatter___date], order: DESC}
+    sort: {frontmatter: {date: DESC}}
```

Comment:
is this the new way of writing query?

Reply:
Yes this is new way for writing graphql queries.